src/main/java/slimeknights/tconstruct/smeltery/tileentity/CastingTableTileEntity.java

```diff
@@ -0,0 +1,17 @@
+package slimeknights.tconstruct.smeltery.tileentity;
+
+import net.minecraft.item.crafting.IRecipeType;
+import net.minecraft.tileentity.TileEntityType;
+import slimeknights.tconstruct.library.recipe.RecipeTypes;
+import slimeknights.tconstruct.smeltery.TinkerSmeltery;
+import slimeknights.tconstruct.smeltery.recipe.AbstractCastingRecipe;
+
+public class CastingTableTileEntity extends AbstractCastingTileEntity {
+
+  public CastingTableTileEntity() {
+    this(TinkerSmeltery.table.get(), RecipeTypes.CASTING_TABLE);
+  }
+  public CastingTableTileEntity(TileEntityType<?> tileEntityTypeIn, IRecipeType<AbstractCastingRecipe> recipeType) {
```

Comment:
Remove this constructor, if you are going to call with existing types you should call the abstract.